OSDOCS-10010: Agent basic install procedure (#73553)
openshift/openshift-docs: modules/installing-ocp-agent-basic-inputs.adoc

// Module included in the following assemblies:
//
// * installing/installing_with_agent_based_installer/installing-with-agent-basic.adoc

:_mod-docs-content-type: PROCEDURE
[id="installing-ocp-agent-basic-inputs_{context}"]
= Creating the configuration inputs

> Use this procedure to create the configuration inputs used to create the agent image.

Reviewer: ```suggestion
You create the configuration inputs to create the agent image.
```

- Same comment about self-ref 

Does "configuration inputs used to create" mean -- "create the configuration inputs to create the agent image." ?

Author: It means you create configuration files that the installer program will consume when it builds the Agent image, I can try wording it more clearly (basically all of the content in this PR was copied from the existing Agent installer docs)

<2> Required. Specify your cluster name.
<3> The cluster network plugin to install. The default value `OVNKubernetes` is the only supported value.
<4> Specify your platform.
+
[NOTE]
====
For bare metal platforms, host settings made in the platform section of the `install-config.yaml` file are used by default, unless they are overridden by configurations made in the `agent-config.yaml` file.
====
<5> Specify your pull secret.
<6> Specify your SSH public key.
<7> Provide the contents of the certificate file that you used for your mirror registry.
The certificate file can be an existing, trusted certificate authority or the self-signed certificate that you generated for the mirror registry.
You must specify this parameter if you are using a disconnected mirror registry.
<8> Provide the `imageContentSources` section according to the output of the command that you used to mirror the repository.
You must specify this parameter if you are using a disconnected mirror registry.
+
[IMPORTANT]
====
* When using the `oc adm release mirror` command, use the output from the `imageContentSources` section.

> * When using `oc mirror` command, use the `repositoryDigestMirrors` section of the `ImageContentSourcePolicy` file that results from running the command.

Reviewer: ```suggestion
* When using the `oc mirror` command, use the `repositoryDigestMirrors` section of the `ImageContentSourcePolicy` file that results from running the command.
```

* `ImageContentSourcePolicy` is deprecated.
====
--

. Create the `agent-config.yaml` file by running the following command:
+
[source,terminal]
----
$ cat > agent-config.yaml << EOF
apiVersion: v1beta1
kind: AgentConfig
metadata:
  name: sno-cluster
rendezvousIP: fd2e:6f44:5dd8:c956::50 // <1>
EOF
----
+
<1> This IP address is used to determine which node performs the bootstrapping process as well as running the `assisted-service` component.

> You must provide the rendezvous IP address when you do not specify at least one host's IP address in the `networkConfig` parameter. If this address is not provided, one IP address is selected from the provided hosts' `networkConfig`.

Reviewer: ```suggestion
You must provide the rendezvous IP address when you do not specify at least one host IP address in the `networkConfig` parameter. If this address is not provided, one IP address is selected from the provided `networkConfig` configuration of hosts.
```

- IMO, it's fine to _replace/host's IP/host IP_ 
See, https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives
> Do not use possessive ’s with inanimate objects.
